### PR TITLE
Intermediate dplyr

### PR DESCRIPTION
**Pull request by SaraTomiolo**
created a partial draft of the intermediate lesson

### CODE REVIEW

File: content/material/2020-01-17-intermediate-dplyr.Rmd [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+The functions we want to use for these operations are `pivot_longer()` and `pivot_wider()` from the tidyr package (Those of you already familiar with this package may know these functions as spread() and gather(). Although these two functions still work, they are being slowly replaced by the pivot_** functions).
+
+To show how these functions work, we are going to use the built-in dataframe `population`.
+
+#### Familiarize with the dataframe
+
+Using the function View() inspect the dataframe population, which shows the populations census in different countries across multiple years.
+
+```{r}
```
Comment (lwjohnst86) [inline content/material/2020-01-17-intermediate-dplyr.Rmd:81]:
```suggestion
```{r, eval=FALSE}
```

File: content/material/2020-01-17-intermediate-dplyr.Rmd [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+### Prerequisites
+
+In this lesson we assume a bit of familiarity with the basic functions of dplyr (especially group-by() and summarize()), which we covered in the [intro_dplyr lesson](https://au-cru.github.io/site/material/2019-11-15-intro-dplyr/) 
+
+
+### Objectives
+
+1. To learn to restructure your data between long and wide format with pivot_longer() and pivot_wider() functions from tidyr package.
+2. Learn to combine group_by() and summarize() function with pivot_longer() and pivot_wider() functions.
```
Comment (lwjohnst86) [inline content/material/2020-01-17-intermediate-dplyr.Rmd]: Put backticks around code in text. E.g.

    `group_by()`

File: content/material/2020-01-17-intermediate-dplyr.Rmd [hunk 3]
```diff
@@ -0,0 +1,333 @@
+---
+title: "2020-01-19-intermediate-dplyr"
+author: 
+- "Sara Tomiolo"
+- "Luke Johnston"
+
```
Comment (lwjohnst86) [inline content/material/2020-01-17-intermediate-dplyr.Rmd]: Add Johannes here.

File: content/material/2020-01-17-intermediate-dplyr.Rmd [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+The functions we want to use for these operations are `pivot_longer()` and `pivot_wider()` from the tidyr package (Those of you already familiar with this package may know these functions as spread() and gather(). Although these two functions still work, they are being slowly replaced by the pivot_** functions).
+
+To show how these functions work, we are going to use the built-in dataframe `population`.
+
+#### Familiarize with the dataframe
+
+Using the function View() inspect the dataframe population, which shows the populations census in different countries across multiple years.
+
+```{r}
+View(population)
```
Comment (lwjohnst86) [inline content/material/2020-01-17-intermediate-dplyr.Rmd:87]: View should not be used within an R Markdown document.

File: content/material/2020-01-17-intermediate-dplyr.Rmd [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+```
+
+
+### Pivot_wider()
```
Comment (lwjohnst86) [inline content/material/2020-01-17-intermediate-dplyr.Rmd]: A bit more descriptive header titles would make it easier to understand the content of the section.

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Thanks so much!! I'll look this over more later, but I have a few comments.

:grinning:

File: content/material/2020-01-17-intermediate-dplyr.Rmd (hunk 3 above)
Comment (SaraTomiolo) [reply content/material/2020-01-17-intermediate-dplyr.Rmd]:
```suggestion
- Johannes Enevoldsen
```

**Review by lwjohnst86 — APPROVED** [review]
Thanks @SaraTomiolo. I'll merge this PR in and address more of the comments in @JohannesNE PR.

**Pull request merged**